msr/msre/pipe_cardinal/gpuCardinal

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+[ $# -ne 4 ] && echo "gpuCardinal requires 4 arguments: %job_name% %number_of_gpus% %runtime_hours% %runtime_minutes%" && exit 0;
+
+envfile="/beegfs/scratch/fangj/Cardinal/Scripts/cardinal_env"
```

Comment:
I moved the settings from environment file `cardinal_env` to the job submission script to avoid confusion. What `cardinal_env` does is basically to define the necessary PATH variables, compilers and modules.